build/gulp/plugins/gulp-component-menu-behaviors.ts

```diff
@@ -0,0 +1,97 @@
+import * as gutil from 'gulp-util'
+import * as path from 'path'
+import * as through2 from 'through2'
+import * as Vinyl from 'vinyl'
+import * as _ from 'lodash'
+import * as fs from 'fs'
+
+const pluginName = 'gulp-component-menu-behaviors'
+const extract = require('extract-comments')
+const doctrine = require('doctrine')
+
+type BehaviorMenuItem = {
+  displayName: string
+  type: string
+  variations: {
+    name: string
+    text: string
+  }
+}
+
+export default () => {
+  const result: BehaviorMenuItem[] = []
+  function bufferContents(file, enc, cb) {
+    if (file.isNull()) {
+      cb(null, file)
+      return
+    }
+
+    if (file.isStream()) {
+      cb(new gutil.PluginError(pluginName, 'Streaming is not supported'))
+      return
+    }
+
+    try {
+      const absPath = path.resolve(process.cwd(), file.path)
+      const dir = path.dirname(absPath)
+      const componentType = _.lowerFirst(path.basename(path.dirname(dir)).replace(/s$/, ''))
+      const behaviorVariantName = file.basename
+      const behaviorName = path.basename(dir)
+
+      let description
+      const fileContent = fs.readFileSync(file.path).toString()
+      const blockComments = extract(fileContent).filter(comment => comment.type === 'BlockComment') // filtering only block comments
+
+      // getting object that describes '@description' part of the comment's text
+      if (!_.isEmpty(blockComments)) {
+        const commentTokens = doctrine.parse(blockComments[0].raw, { unwrap: true }).tags
+        const descriptionToken = commentTokens.find(token => token.title === 'description')
+        description = descriptionToken.description
```

Comment:
Check `descriptionToken` is defined. Now if there is a block comment without `@description` build crashes.